#This is just to say that oah this is a shell script not a bash
# we can add the bash dependency but its not a good idea
diff --git a/src/efivarfs/efivarfs.sh b/src/efivarfs/efivarfs.sh
old mode 100644
new mode 100755
index 77edcdc..7d64ff0
--- a/src/efivarfs/efivarfs.sh
+++ b/src/efivarfs/efivarfs.sh
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/bin/sh
 
 efivarfs_mount=/sys/firmware/efi/efivars
 test_guid=210be57c-9849-4fc7-a635-e6382d1aec27
